axelar-chains-config/info/testnet.json

```
@@ -2127,6 +2127,23 @@
         "url": "https://sepolia-explorer.base.org",
         "api": "https://api-sepolia.basescan.org/api"
       }
+    },
+    "blast-sepolia": {
+      "name": "Blast Sepolia",
```

```suggestion
      "name": "Blast-Sepolia",
```
to avoid whitespace in this config. also missing for Base above

---

Updating the names for Blast sepolia and Base sepolia, Not updating this in [Chains repo](https://github.com/axelarnetwork/chains/blob/5b688fc45ccf7340a28ae30c5495e91ceb933bdc/testnet.py#L136) as all sepolia chains in that repo contain space in their names.